content/en/graphing/guide/dashboard-lists-api-doc.md

```diff
@@ -0,0 +1,941 @@
+---
+title: Dashboard Lists API doc (v1)
+kind: guide
+aliases:
+  - /graphing/faq/dashboard-lists-api-doc
+---
+
+<div class="alert alert-danger">
+This endpoint is outdated. Use the <a href="https://docs.datadoghq.com/api/?lang=python#dashboard-lists">new Dashboard List endpoint version</a> instead.
+</div>
+
+Interact with your dashboard lists through the API to make it easier to organize, find, and share all of your dashboards with your team and organization.
+
+## Get Items of a Dashboard List
+
+### Signature
+
+`GET https://api.datadoghq.com/api/v1/dashboard/lists/manual/<LIST_ID>/dashboards`
+
+### Examples
+
+{{< tabs >}}
+{{% tab "Python" %}}
+
+##### Example Request
+
+```python
+from datadog import initialize, api
+
+options = {
+    'api_key': '<YOUR_API_KEY>',
+    'app_key': '<YOUR_APP_KEY>'
+}
+
+initialize(**options)
+
+api.DashboardList.get_items(4741)
+
+```
+
+##### Example Response
+
+```python
+{
+    'total': 2,
+    'dashboards': [
+        {
+            'new_id': 99,
```

Comment:
`new_id` uses the new format. The same as `id` in `v2.  For example: `rys-xwq-geh`.